htlcswitch/switch_test.go

```diff
@@ -74,9 +74,7 @@ func TestSwitchAddDuplicateLink(t *testing.T) {
 	}
 
 	// Remove the live link to ensure the indexes are cleared.
-	if err := s.RemoveLink(chanID1); err != nil {
-		t.Fatalf("unable to remove alice link: %v", err)
-	}
+	s.RemoveLink(chanID1)
```

Comment:
Why are we no longer checking the return value here?

Reply:
The semantics are now such that it can't fail, it will block until all state related to a channel is removed or doesn't exist. The new function signature on `RemoveLink` doesn't return anything.